REFACTOR.md

```diff
@@ -0,0 +1,18 @@
+# Refactoring Baikal to the Silex framework
+
+- Original webroot is now at 'html.old'
+- New webroot remains the same 'html' for easier upgrades
+- Composer autoloader switch to psr-4 and support both locations
```

Comment:
I kind of agree, but in this case it was just a small composer-config change, and not a drastic amount of changes in file paths, so I think it's ok.

Besides that, I think there's no way we can make this PR short-lived :( it's a major overhaul